src/Compilers/CSharp/Test/Semantic/Diagnostics/CompilationEventTests.cs

```diff
@@ -94,10 +94,15 @@ partial class C<T1>
                 .VerifyDiagnostics();
             VerifyEvents(q,
                 "CompilationStartedEvent",
+                "SymbolDeclaredCompilationEvent(System System)",
```

Comment:
>"SymbolDeclaredCompilationEvent(System System)" [](start = 16, length = 47)

Should events for implicitly declared symbols be suppressed? Are we generating events for other synthesized symbols such as the backing field for `P`?

Reply:
I'll add a PROTOTYPE.
We're not generating an event for backing fields or embedded attributes, but their are not accessible in source, whereas injected attributes are. So I'm not sure what is the expectation.
Do you know who could advise on API behavior?

---
In reply to: [214441403](https://github.com/dotnet/roslyn/pull/29180#discussion_r214441403) [](ancestors = 214441403)